Assignments/discord/KCreel/persiflage/src/App.js

```diff
@@ -0,0 +1,163 @@
+import React, {useState} from 'react';
+import './App.css';
+import { Grid, Container, List } from 'semantic-ui-react';
+
+const uuidv1 = require('uuid/v1');
+//[items, setItem] = useState(
+const Server = ({ servers, setCurrentServer }) => {
+  return (
+    <Grid.Column>
+     {servers.map(server =>(
+       <Grid.Row>
+         <span onClick={changeServer(setCurrentServer, server.serverName)}>{server.serverName}</span>
+       </Grid.Row>
+     ))}
+    </Grid.Column>
+  )
+}
+function changeServer(setCurrentServer, newserverName){
+  const serverName = newserverName;
+  setCurrentServer(serverName);
+}
+const SignIn =({user, setUser}) =>{
+  if(user == '') return (
+    <input id='userNameBox'placeholder='Username:' onKeyPress={function(event){changeUser(event, user, setUser)}}/>
+  )
+  //2.0   if(event.which==="13"){setUser(this.value)}
+  //onChange={(event)=>{changeUser(event, setCurrentUser)}}
+  return (<div>Welcome, {user}</div>)
+  
+}
+function changeUser(event, user, setUser){
+  console.log("The keycode is:"+event.which);
+  if(event.which=="13") {
```

Comment:
could turn this into a `const` since you use it later in code
`const RETURN_KEY = '13'`